examples/api-tests/src/file-search.spec.js

```diff
@@ -55,6 +55,62 @@ describe('file-search', function () {
 
         });
 
+        describe('#filterAndRange', () => {
+
+            it('should return the default when not searching', () => {
+                const filterAndRange = quickFileOpenService['filterAndRange'];
+                assert.equal(filterAndRange, quickFileOpenService['filterAndRangeDefault']);
+            });
+
+            it('should update when searching', () => {
+                quickFileOpenService['onType']('a:2:1', () => { }); // perform a mock search.
+                const filterAndRange = quickFileOpenService['filterAndRange'];
+                assert.equal(filterAndRange.filter, 'a');
+                assert.deepEqual(filterAndRange.range, { start: { line: 1, character: 0 }, end: { line: 1, character: 0 } });
+            });
+
+        });
+
+        describe('#splitFilterAndRange', () => {
+
+            const expression1 = 'a:2:1';
+            const expression2 = 'a:2,1';
+            const expression3 = 'a:2#2';
+            const expression4 = 'a#2:2';
+            const expression5 = 'a#2,1';
+            const expression6 = 'a#2#2';
+            const expression7 = 'a:2';
+            const expression8 = 'a#2';
+
+            it('should split the filter correctly for different combinations', () => {
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression1).filter), 'a');
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression2).filter), 'a');
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression3).filter), 'a');
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression4).filter), 'a');
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression5).filter), 'a');
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression6).filter), 'a');
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression7).filter), 'a');
+                assert.equal((quickFileOpenService['splitFilterAndRange'](expression8).filter), 'a');
+            });
+
+            it('should split the range correctly for different combinations', () => {
+
+                const rangeTest1 = { start: { line: 1, character: 0 }, end: { line: 1, character: 0 } };
+
```

Comment:
No need for this extra line :laughing: :see_no_evil:

Reply:
hahahaha alright sorry